src/Uno.UI/Controls/Window.iOS.cs

```diff
@@ -352,14 +366,15 @@ internal void MakeVisible(UIView view, BringIntoViewMode? bringIntoViewMode, boo
 			}
 
 			var scrollView = view.FindSuperviewsOfType<UIScrollView>().LastOrDefault();
-			_scrollViewModifiedForKeyboard = new WeakReference<UIScrollView>(scrollView);
```

Comment:
do we want to clear the previous WeakReference if the new one is null. Looking at it now I guess that the answer is Yes.

Reply:
I would say yes too